frontend/src/metabase/collections/components/BaseTableItem.tsx

```diff
@@ -58,16 +69,17 @@ export function BaseTableItem({
   onMove,
   onDrop,
   onToggleSelected,
-}) {
+}: BaseTableItemProps) => {
   const handleSelectionToggled = useCallback(() => {
-    onToggleSelected(item);
+    onToggleSelected?.(item);
   }, [item, onToggleSelected]);
 
   const renderRow = useCallback(() => {
     const canSelect =
-      collection.can_write && typeof onToggleSelected === "function";
+      collection?.can_write && typeof onToggleSelected === "function";
 
-    const lastEditInfo = item["last-edit-info"];
+    const lastEditInfo =
+      "last-edit-info" in item ? item["last-edit-info"] : undefined;
```

Comment:
what does this do that the previous code didn't?

Reply:
Good catch